frontend/packages/ceph-storage-plugin/src/components/dashboards/persistent-internal/status-card/status-card.tsx

```diff
@@ -74,6 +81,72 @@ export const StatusCard: React.FC<DashboardItemProps> = ({
     t,
   );
 
+  const [isHCPresent, setHCPresence] = React.useState(false);
+  const healthChecksGetterOrPresent = (checkPresence: boolean = false) => {
+    const cephDetails = data?.[0]?.status?.ceph?.details;
+    const pattern = /[A-Z]+_*|error/g;
+    const healthChecks: CephHealthCheck[] = [];
+    for (const key in cephDetails) {
+      if (cephDetails.hasOwnProperty(key) && pattern.test(key)) {
+        if (checkPresence) {
+          return true;
+        }
+        const healthCheckObject: CephHealthCheck = {
+          details: cephDetails[key].message,
+          troubleshootLink: whitelistedHealthChecksRef[key] ?? null,
+        };
+        healthChecks.push(healthCheckObject);
+      }
+    }
+    return checkPresence ? false : healthChecks;
+  };
+
+  React.useEffect(
+    () => {
+      setHCPresence(healthChecksGetterOrPresent(true) as boolean);
+    },
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+    [data?.[0]?.status?.ceph?.details],
+  );
+
+  const CephHealthChecks: React.FC = () => {
+    const healthChecks = healthChecksGetterOrPresent();
+    const getClassName = () =>
+      (healthChecks as CephHealthCheck[]).length > 5 ? 'healthchecks-scrollable' : null;
+    return (
+      <>
+        <hr className="stylize-hr" />
```

Comment:
```suggestion
        
```
CSS will takecare

Reply:
Introducing an `<hr/>` was Hadas' suggestion in a sync yesterday. Basically the scrollbar kind of looked odd just by itself so this was suggested to differentiate between the headers and content a bit more cleanly, I think.